src/lib/tenant/configs/ui/ens.ts

```diff
@@ -87,6 +87,22 @@ export const ensTenantUIConfig = new TenantUI({
       name: "snapshotVotes",
       enabled: true,
     },
+    {
+      name: "sponsoredVote",
+      enabled: true,
+      config: {
+        domain: "ENS Governor",
+        version: "1",
+      },
+    },
+    {
+      name: "sponsoredDelegate",
+      enabled: true,
+      config: {
+        domain: "Ethereum Name Service",
```

Comment:
We will need to update this to handle different contract names between Sepolia and Mainnet.